doc/nrf/ug_matter_overview_commissioning.rst

```diff
@@ -0,0 +1,157 @@
+
+Establishing fail-safe
+======================
+
+The commissioner requests the commissionee to back up its original configuration.
+The fail-safe acts as a back-up, but it also starts a timer that sets a limit for the whole commissioning process.
+The timer is disabled with the disarming of the fail-safe at the end of commissioning.
+
+Preliminary node configuration
+==============================
+
+The commissioner reads the Basic Cluster Information of the commissionee and its device type.
+It then configures the commissionee with regulatory information, such as location and country, and the current UTC time.
+
+Device Attestation Certificate verification
+===========================================
+
+The commissioner checks whether the commissionee is a certified Matter device.
+As part of this verification, the commissioner generates a random 32-bit attestation nonce and sends it to the commissionee, who should return the signed attestation information that includes the nonce.
+Usage of a nonce prevents replay attacks against commissioners.
+The commissioner then validates the attestation information.
+
+The verification succeeds if the device is able to prove the validity and ownership of the mandatory :ref:`ug_matter_device_attestation` elements.
+If the validity and ownership cannot be proven, the verification fails.
+The commissioner can then either terminate or continue the commissioning procedure.
+
+Installing operational credentials
+==================================
+
+The commissioner installs Node Operational Certificate (NOC) and Operational ID on the commissionee.
+The commissionee becomes the new node of the Matter fabric.
+The node is identified by a tuple consisting of the Root PK, Fabric ID, and Node ID.
+(While the fabric is identified by a tuple consisting of the Root PK and the Fabric ID.)
+
+Network commissioning
+=====================
+
+The commissioner provisions the commissionee node with the operational network credentials, either Wi-Fi or Thread, and requests the commissionee to connect to the network.
+
+Operational discovery
+=====================
+
+The commissioner discovers the commissionee node on the operational network using DNS-SD.
+This way, the commissioner learns the IP address of the node.
+
+Security setup with CASE
+========================
+
+The commissioner and the node use the Certificate-Authenticated Session Establishment (CASE) protocol to establish secure communication.
+The CASE protocol is in charge of exchanging NOCs to set up a session secured with a new pair of keys.
+The CASE connection is reset each time a device breaks the connection.
+
+Disarming fail-safe
+===================
+
+The commissioner requests the commissionee node to remove the stored configuration backup.
+This also stops the fail-safe timer.
+
+Message exchange
+================
+
+The commissioner and the commissionee exchange an AES-encrypted message on the operational network.
```

Comment:
I guess this was meant to indicate that now the devices may normally talk to each other so it's not a single message. They can now exchange a number of messages.

Reply:
Rephrased to "The commissioner and the commissionee start exchanging AES-encrypted messages on the operational network."